lib/AST/ASTContext.cpp

```diff
@@ -3951,8 +3951,17 @@ bool ASTContext::isStandardLibraryTypeBridgedInFoundation(
           nominal == getStringDecl() ||
           nominal == getErrorDecl() ||
           nominal == getAnyHashableDecl() ||
-          // Weird one-off case where CGFloat is bridged to NSNumber.
-          nominal->getName() == Id_CGFloat);
+          // Foundation is a dependency of CoreGraphics, but
```

Comment:
@atrick or @aschwaighofer, how big of a perf impact do you think it'd be if we assumed all imported C value types might have an `_ObjectiveCBridgeable` conformance?